src/main/java/org/kohsuke/github/GHProjectCard.java

```diff
@@ -125,11 +125,16 @@ public GHIssue getContent() throws IOException {
         if (StringUtils.isEmpty(content_url))
             return null;
         try {
+            GHIssue issue;
             if (content_url.contains("/pulls")) {
-                return root().createRequest().withUrlPath(getContentUrl().getPath()).fetch(GHPullRequest.class);
+                issue = root().createRequest().withUrlPath(getContentUrl().getPath()).fetch(GHPullRequest.class);
             } else {
-                return root().createRequest().withUrlPath(getContentUrl().getPath()).fetch(GHIssue.class);
+                issue = root().createRequest().withUrlPath(getContentUrl().getPath()).fetch(GHIssue.class);
             }
+            GHRepository repository =
+                root().createRequest().withUrlPath(issue.getRepositoryUrlPath()).fetch(GHRepository.class);
+            issue.wrap(repository);
```

Comment:
Perhaps this would be better as part of `GHIssue.getRepository()`?  That way clients only request the repo if they need it. 

```suggestion
            GHRepository repository =
                root().createRequest().withUrlPath(issue.getRepositoryUrlPath()).fetch(GHRepository.class);
            issue.wrap(repository);
```

Reply:
I can't put this inside of `getRepository` without changing the signature of the method. Calling `fetch` throws a checked `IOException`. So if I were to put this logic inside of `getRepository` then that method would also need to throw an `IOException`.

Reply:
If I'm allowed to make API breaking changes, I'm happy to do this the way you propose.